.github/workflows/build-test.yaml

```diff
@@ -1788,6 +1788,41 @@ jobs:
           aws-secret-access-key: '${{ secrets.E2E_SUPPORT_BUNDLE_AWS_SECRET_ACCESS_KEY }}'
 
 
+  validate-airgap-smoke-test:
+    runs-on: ubuntu-20.04
+    needs: [ enable-tests, can-run-ci, build-kots, build-e2e, build-kurl-proxy, build-migrations, push-minio, push-rqlite ]
+    strategy:
+      fail-fast: false
+      matrix:
+        k8s_version: [ v1.24.4-k3s1 ]
```

Comment:
hmm, we might need to run this on the latest 3-4 supported kubernetes versions like the online smoke test. maybe it's not necessary with the current test implementation, but it might be relevant in the future as this test gets more comprehensive.